packages/widget-v2/src/pages/SwapExecutionPage/SwapExecutionPageRouteSimpleRow.tsx

```diff
@@ -49,13 +47,6 @@ export const SwapExecutionPageRouteSimpleRow = ({
     tokenAmount,
   });
 
-  const txStateOfAnimatedBorder = useMemo(() => {
-    if (destination && status === "broadcasted") {
-      return;
-    }
-    return status;
-  }, [status, destination]);
-
```

Comment:
broadcasted would not show any loading animation in animated border, this code is unncessary/redundant